setup.json

```diff
@@ -6,7 +6,7 @@
             "show": true,
             "pos": "left",
             "args": {
-                "masterPath": "/home/rabi/artiq/"
+                "masterPath": "/home/jaehun/quiqcl/artiq/examples/"
```

Comment:
Maybe this is mistakenly staged here.
You can make another setup file (e.g. "setup_jaehun.json") and ignore it in your git repo.

Reply:
Oh, I made a mistake..
Thanks for offering a solution.